### ISSUE TITLE
Lastfm Server No Longer Exists

### ISSUE THREAD

**Issue by interfect** [lastfm/Fingerprinter#11]
This tool no longer appears to work:

```
$ ./lastfm-fpclient ~/Music/Bit\ Shifter/Information\ Chase/01\ -\ Bit\ Shifter\ -\ Chase\ Init.mp3 
ERROR: Empty reply from server$
```

Presumably Last.fm has either discontinued their fingerprinting API, or changed the way it works without updating the client.

This is potentially a duplicate of #9, but if it;s been broken since that was posted, presumably it's supposed to be broken.

**Comment by Eiriksmal** [comment]
Yeah, they've shut down their fingerprinting API, as far as I can tell. The tool hasn't worked for more than a year or so.
